docs/user_guide/mtf_linters.rst

```diff
@@ -1,8 +1,70 @@
 Linters
 =================
 
-MTF provides a set of linters for checking containers
+MTF provides a set of linters for checking containers, help files and dockerfiles.
 
 Dockerfile linters
 ~~~~~~~~~~~~~~~~~~
+Dockerfile linters are divided into two python files: `dockerlint.py`_ and `dockerfile_lint.py`_.
+
+.. _dockerlint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/generic/dockerlint.py
+.. _dockerfile_lint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/static/dockerfile_lint.py
+
+`dockerlint.py`_ performs these checks on an existing container image:
+
+* **test_all_nodocs** checks if documentation files shipped by an install RPM packages. There are installed in the ``base image`` or by ``RUN`` commands." . This is only ``WARN`` check.
+* **test_installed_docs** checks if RPM packages installed by ``RUN dnf`` command installs also documentation files. Except ``base image``.
```

Comment:
s/by ``RUN dnf`` command installs also/by the ``RUN dnf`` command also installs/

s/Except base image/The base image is an exception./